docs/USAGE-GUIDE.md

```diff
@@ -1113,6 +1113,10 @@ connection. **s2n_recv** will return the number of bytes read and also return
 
 **NOTE:** Unlike OpenSSL, repeated calls to **s2n_recv** should not duplicate the original parameters, but should update **buf** and **size** per the indication of size read. For example;
 
+**NOTE:** By default, **s2n_recv** will return after reading a single TLS record. To change this
+behavior such that it will read up to **size**, the config for the connection can be updated
+by calling **s2n_config_enable_recv_multi_record**.
```

Comment:
I'm trying to think _how_ an application would break on this and if it's worth it to just turn on by default... An application that does expect record-sized messages in response to a single recv seems already broken by design since the peer can frame payloads however it likes.

Reply:
I agree, doing it this way is safer - as it turns out some s2n tests expect this behavior. While we can fix it and change the default, it may have unintended consequences, and as such we should give applications the chance to fix themselves.

Reply:
I guess there's always s2n-tls v2 😢 ...